src/main/java/com/knu/buga1chuk/serialization/service/PersonBinaryService.java

```diff
@@ -45,4 +48,8 @@ public List<Person> getPersonFromFile(File file) {
         return personList.getPersons();
     }
 
+    @Override
+    public Person toPerson(String input) {
+        return null;
```

Comment:
It is not good to return null.
Check the documentation of the unsupported exception.

Reply:
Fix